### PR TITLE
Add support for java runtime

### PR DESCRIPTION
**Pull request by jtfell**
Functions with runtime set to `java` are delegated back to the vanilla package plugin rather than kicking off a webpack build.

### CODE REVIEW

File: src/ServerlessBuildPlugin.ts [hunk 1]
```diff
@@ -239,12 +240,43 @@ export class ServerlessBuildPlugin {
     }
   }
 
+  async buildFunction (fnName, fnConfig) {
+
+    const runtime = (fnConfig.runtime && fnConfig.runtime) || this.serverless.config.service.runtime;
```
Comment (nfour) [inline src/ServerlessBuildPlugin.ts]: Whats with `(fnConfig.runtime && fnConfig.runtime)`

File: src/ServerlessBuildPlugin.ts [hunk 2]
```diff
@@ -239,12 +240,43 @@ export class ServerlessBuildPlugin {
...
+    if (runtime.indexOf('java') > -1) {
+      return this.buildJavaFunction(fnName, fnConfig);
+    }
+
+    throw new this.serverless.classes.Error(
+      '[serverless-build-plugin]: Unsupported runtime. Only supports nodejs and java.',
+    );
+  }
+
+  async buildJavaFunction (fnName, fnConfig) {
```
Comment (nfour) [inline src/ServerlessBuildPlugin.ts]: Considering this does nothing, you should call this `passthroughFunction`

File: test/1.0/node_modules/serverless-build-plugin [hunk 3]
```diff
@@ -1 +0,0 @@
-../../../dist
```
Comment (nfour) [inline test/1.0/node_modules/serverless-build-plugin]: What changed here ???

File: src/ServerlessBuildPlugin.ts (hunk 1 above)
Comment (jtfell) [reply src/ServerlessBuildPlugin.ts]: yeah oops, I'll fix that

File: src/ServerlessBuildPlugin.ts (hunk 2 above)
Comment (jtfell) [reply src/ServerlessBuildPlugin.ts]: Yep sure.

File: test/1.0/node_modules/serverless-build-plugin (hunk 3 above)
Comment (jtfell) [reply test/1.0/node_modules/serverless-build-plugin]: I'm not sure, I couldn't revert it for some reason. I don't think it actually changes anything

**Pull request merged**